### ISSUE TITLE
Disable actions on Portal if RPC not connected

### ISSUE THREAD

**Issue by prayagd** [pendulum-chain/portal#426]
As a user i should not able to do any action on the Portal if the RPC is not connected

## Acceptance criteria
* If RPC endpoint is not successfully connected to the Portal disable actions on Portal
* Show a dialog box with message saying "RPC connection unsuccessful, Try refreshing"
* User should be able to close this dialog box

**Comment by prayagd** [comment]
@Sharqiewicz would it be helpful for you if there are designs for this?

**Comment by ebma** [comment]
What does 'disable actions' mean? This is too ambiguous for me and we could implement it in different ways. So having designs would be nice so we are all aligned how it's supposed to look.

**Comment by vadaynujra** [comment]
@prayagd for better clarity, could you share screenshots / video of what this ticket would additionally disable, given that user cannot do much on the portal anyway in case the RPC is down?

**Comment by prayagd** [comment]
This is how it should look like, the message keeps appearing until the user is connected to RPC.   

<img width="1413" alt="Screenshot 2024-08-22 at 1 08 00 PM" src="https://github.com/user-attachments/assets/7ea4296a-e653-4cb1-902f-86f7704472d5">

**Comment by Sharqiewicz** [comment]
Why is it blocked @prayagd ?

**Comment by prayagd** [comment]
Not anymore @Sharqiewicz earlier there were open comments

**Comment by ebma** [comment]
I would like to suggest the following changes:
- Make the dialog show the text: "Couldn't establish a connection to the Pendulum RPC nodes. Please try reloading the page. If this issue persists, please reach out to us here (<insert link to somewhere, probably discord>)"
- Add a button "Reload page" below the text.

**Comment by prayagd** [comment]
<img width="1393" alt="Screenshot 2024-08-27 at 12 39 59 PM" src="https://github.com/user-attachments/assets/25300b3e-9ea0-46e9-ae3f-fdb2c4e547b0">

something like this?

**Comment by ebma** [comment]
Yes exactly 👍 We don't have to keep the exact wording but this way it prominently features the action the user should do (ie reload) and it sounds less like there is a major issue on our infrastructure.
